frontend/src/components/keyboard/keyboardSettings.js

```diff
@@ -56,7 +56,7 @@ export const buttonTheme = [
   },
   {
     class: 'incline-buttons',
-    buttons: 'd f',
```

Comment:
Why we removed [d-f]? New icons reflect [d-f] changes. But old icons should be moved to [a s] until we get new versions.